app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
seems fine to me!

btw, it might be nice to offer multiple formats (e.g. `MATERIAL_COLOR_CYAN_500` and `$material-color('cyan', '500');`) if you use multiple environments